qutebrowser/browser/hints.py

```diff
@@ -137,6 +140,24 @@ class HintManager(QObject):
         Target.spawn: "Spawn command via hint",
     }
 
+    FIRST_ALPHABETIC = re.compile('[A-Za-z]{3,}')
+
+    TAG_EXTRACTORS = dict(
+        alt=lambda elem: elem["alt"],
+        name=lambda elem: elem["name"],
+        title=lambda elem: elem["title"],
+        src=lambda elem: elem["src"].split('/')[-1],
+        href=lambda elem: elem["href"].split('/')[-1],
+        text=str,
+    )
+
+    TAGS_FOR = collections.defaultdict(
+        list, {
+            "IMG": ["alt", "title", "src"],
+            "A": ["title", "href", "text"],
+            "INPUT": ["name"]
+        })
+
```

Comment:
As you need all those in one place only, I'm not sure if it makes sense to define them as class constants - I think I'd prefer them being local, what do you think?